webpack.config.js

```diff
@@ -36,6 +36,14 @@ module.exports = {
         test: /\.(css|scss)?$/,
         use: ['style-loader', 'css-loader', 'sass-loader'],
       },
+      {
+        test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
```

Comment:
what's this for? just the little icons?

Reply:
yes its for the little arrow icons that come with the Datetimepicker